README.md

```diff
@@ -35,6 +35,15 @@ paas-admin:
 	redirect-uri: "https://[pass-admin-domain.com]/auth/login/callback"
 ```
 
+If you get problems with "Invalid redirect", use uaac to modify the
+redirect-uri:
+
+```
+uaac target https://uaa.my.environment
+uaac token client get admin -s my-uaa-admin-client-secret
+uaac client update paas-admin --redirect-uri http://localhost:3000/auth/login/callback
```

Comment:
Do you think it's worth actually including other attributes from the above? It may take a while before these changes make it to `paas-cf`.

Reply:
Can you clarify what you mean by including other attributes?

Reply:
as in:

```sh
bundle exec uaac client update \
        --name ${CLIENT_NAME} \
        --scope scim.me,openid,profile,uaa.user,cloud_controller.read,cloud_controller.admin,cloud_controller.admin_read_only,cloud_controller.global_auditor \
        --authorized_grant_types authorization_code,client_credentials \
        --authorities uaa.none,scim.invite,cloud_controller.write \
        -s ${CLIENT_SECRET} \
        --autoapprove true \
        --redirect_uri http://localhost:3000/auth/login/callback \
        ${CLIENT_NAME}
```